src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -82,15 +82,15 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
 
   private void writeAvailableLines() {
     for (String line : stdoutParser.getAvailableLines()) {
-      final String trimmed = line.trim();
-
-      if (trimmed.startsWith("[{") && trimmed.endsWith("}]")) {
-        LOG.info(trimmed);
+      if (DaemonApi.parseAndValidateDaemonEvent(line.trim()) != null) {
+        if (FlutterSettings.getInstance().isVerboseLogging()) {
+          LOG.info(line.trim().trim());
+        }
         return;
       }
       else {
-        // We're seeing a spurious newline before some launches; this removed any single
-        // newline that occur before we've printed text.
+        // We're seeing a spurious newline before some launches; this removed any single newline that occur
```

Comment:
Also not important but something doesn't agree with me.

Maybe:

this removes any single newline that occurs

or 

this removes any single newlines that occur

or

this removes any single newline that occured

Reply:
updated: `this removes any single newline that occurred`